src/test/java/seedu/address/logic/commands/FindStudentCommandTest.java

```diff
@@ -77,7 +77,7 @@ public void execute_multipleKeywords_multipleStudentsFound() {
     /**
      * Parses {@code userInput} into a {@code NameContainsKeywordsPredicate}.
```

Comment:
```suggestion
     * Parses {@code userInput} into a {@code StudentNameContainsKeywordsPredicate}.
```

Reply:
I guess the refactor cant detect that... THANKS!